yarn.lock

```diff
@@ -4661,6 +4735,18 @@
     "@types/uglify-js" "*"
     source-map "^0.6.0"
 
+"@types/webpack@^4.4.31":
```

Comment:
Why are there multiple sets of webpack types now?

Reply:
likely because storybook depends on wp4 and uses that internally. I will check.

Reply:
If we could update storybook too (likely in a different PR) that would be nice, though not sure how much work it would be, and not as high priority.

Reply:
https://github.com/storybookjs/storybook/issues/9216

Reply:
proceed with caution. Storybook is NOT compatible with webpack 5 yet. My team overlooked the issue while upgrading to webpack 5. Now we are stuck with the external issue dependency mentioned above.

Reply:
Indeed, storybook depends on it.

```bash
[ ~/fluentui/b1 ] yarn why webpack
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "webpack"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "webpack@5.12.3"
info Has been hoisted to "webpack"
info Reasons this module exists
   - "workspace-aggregator-7ccba125-3adc-4dc0-92fd-bcb3cd8e3e7a" depends on it
   - Hoisted from "_project_#ssr-tests#webpack"
   - Hoisted from "_project_#@fluentui#web-components#webpack"
   - Hoisted from "_project_#@fluentui#webpack-utilities#webpack"
   - Hoisted from "_project_#@fluentui#scripts#webpack"
   - Hoisted from "_project_#@fluentui#digest#webpack"
   - Hoisted from "_project_#@fluentui#local-sandbox#webpack"
info Disk size without dependencies: "14.05MB"
info Disk size with unique dependencies: "21.77MB"
info Disk size with transitive dependencies: "33.72MB"
info Number of shared dependencies: 79
=> Found "@storybook/core#webpack@4.43.0"
info This module exists because "_project_#@fluentui#scripts#@storybook#core" depends on it.
info Disk size without dependencies: "3.24MB"
info Disk size with unique dependencies: "14.09MB"
info Disk size with transitive dependencies: "42.8MB"
info Number of shared dependencies: 159
=> Found "@storybook/react#webpack@4.43.0"
info This module exists because "_project_#vr-tests#@storybook#react" depends on it.
info Disk size without dependencies: "2.48MB"
info Disk size with unique dependencies: "13.34MB"
info Disk size with transitive dependencies: "42.05MB"
info Number of shared dependencies: 159
=> Found "corejs-upgrade-webpack-plugin#webpack@4.43.0"
info This module exists because "_project_#@fluentui#scripts#@storybook#core#corejs-upgrade-webpack-plugin" depends on it.
info Disk size without dependencies: "2.4MB"
info Disk size with unique dependencies: "13.25MB"
info Disk size with transitive dependencies: "41.96MB"
info Number of shared dependencies: 159
✨  Done in 14.67s.
[ ~/fluentui/b1 ] yarn why corejs-upgrade-webpack-plugin
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "corejs-upgrade-webpack-plugin"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "corejs-upgrade-webpack-plugin@2.2.0"
info Reasons this module exists
   - "_project_#@fluentui#scripts#@storybook#core" depends on it
   - Hoisted from "_project_#@fluentui#scripts#@storybook#core#corejs-upgrade-webpack-plugin"
info Disk size without dependencies: "5.58MB"
info Disk size with unique dependencies: "19.65MB"
info Disk size with transitive dependencies: "39.32MB"
info Number of shared dependencies: 80
✨  Done in 3.66s.
```